<html><head></head><body><pre>--- /home/wxwindows/public_html/cvsdocs/wxWidgets/docs/latex/wx/menuitem.tex	2005-03-10 21:51:37.000000000 +0100
+++ docs/latex/wx/menuitem.tex	2005-08-24 02:22:43.000000000 +0200
@@ -20,7 +20,6 @@
 
 \wxheading{Derived from}
 
-wxOwnerDrawn (Windows only)\\
 \helpref{wxObject}{wxobject}
 
 \wxheading{Include files}
</pre></body></html>
 	  	 
